src/TorchSharp/NN/Sequential.cs

```diff
@@ -5,6 +5,7 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Runtime.InteropServices;
+using TorchSharp.torchvision;
```

Comment:
Why? I would like to avoid circular dependencies between torch.nn and torchvision. The latter should, technically, be separable into its own package some time in the future.